java/ql/lib/semmle/code/java/frameworks/Objects.qll

```diff
@@ -9,8 +9,10 @@ private class ObjectsSummaryCsv extends SummaryModelCsv {
       [
         //`namespace; type; subtypes; name; signature; ext; input; output; kind`
         "java.util;Objects;false;requireNonNull;;;Argument[0];ReturnValue;value",
-        "java.util;Objects;false;requireNonNullElse;;;Argument[0..1];ReturnValue;value",
+        "java.util;Objects;false;requireNonNullElse;;;Argument[0];ReturnValue;value",
+        "java.util;Objects;false;requireNonNullElse;;;Argument[1];ReturnValue;value",
         "java.util;Objects;false;requireNonNullElseGet;;;Argument[0];ReturnValue;value",
+        "java.util;Objects;false;requireNonNullElseGet;;;Argument[1];ReturnValue;value",
```

Comment:
This looks wrong. `Argument[1]` is not being returned by value. This should rather be something like `ReturnValue of Argument[1]`, but that's not quite supported yet.

Reply:
Would you just leave it out then?

Reply:
Yeah, that's what we've been doing so far.

Reply:
You might want to put a comment along the lines of "// Note that when lambdas are supported we should have more models for requireNonNullElseGet" so that we can easily add these models when lambdas are supported. That's what I've been doing with Apache Commons Collections.